gradle/dependencies.gradle

```diff
@@ -102,7 +102,9 @@ libraries += [
     jansi: dependencies.module('org.fusesource.jansi:jansi:1.14'),
     xerces: "xerces:xercesImpl:2.9.1",
     objenesis: 'org.objenesis:objenesis:1.2@jar',
-    jsoup:'org.jsoup:jsoup:1.6.3'
+    jsoup:'org.jsoup:jsoup:1.6.3',
+    jgit: 'org.eclipse.jgit:org.eclipse.jgit:4.8.0.201706111038-r',
+    jgitTest: 'org.eclipse.jgit:org.eclipse.jgit.junit:4.8.0.201706111038-r',
```

Comment:
We should check what these are bringing in transitively (it looks like JGit needs a newer version of HTTPclient than we use currently).

Reply:
I think you might be reading the JGit dependencies incorrectly.
According to [this page](https://mvnrepository.com/artifact/org.eclipse.jgit/org.eclipse.jgit/4.8.0.201706111038-r) jgit uses:

com.googlecode.javaewah » JavaEWAH | 1.1.6
com.jcraft » jsch | 0.1.54
org.apache.httpcomponents » httpclient | 4.3.6
org.slf4j » slf4j-api | 1.7.2

And Gradle already lists in [dependencies.gradle](https://github.com/gradle/gradle/blob/master/gradle/dependencies.gradle) uses:
com.jcraft » jsch | 0.1.54
https://github.com/gradle/gradle/blob/f7b54ff363a20326cd88e3419334d055420bcc1f/gradle/dependencies.gradle#L42
org.apache.httpcomponents » httpclient | 4.4.1
https://github.com/gradle/gradle/blob/f7b54ff363a20326cd88e3419334d055420bcc1f/gradle/dependencies.gradle#L79
org.slf4j » slf4j-api | 1.7.10
https://github.com/gradle/gradle/blob/f7b54ff363a20326cd88e3419334d055420bcc1f/gradle/dependencies.gradle#L50

So, our dependencies are all newer than (or the same as) the ones used by jgit. Maybe you were reading the "Updates" column and interpreting the fact that there was an newer version of a particular library as meaning that jgit was depending on that newer version?

If you're still concerned, I think this build scan should calm any nerves:
jsch: https://scans.gradle.com/s/3luajehcw7w2i/dependencies?dependencies=jsch&expandAll 
httpclient: https://scans.gradle.com/s/3luajehcw7w2i/dependencies?dependencies=httpclient&expandAll
slf4j: https://scans.gradle.com/s/3luajehcw7w2i/dependencies?dependencies=slf4j&expandAll